pkg/agent/client.go

```diff
@@ -502,7 +502,7 @@ func (a *Client) remoteToProxy(connID int64, ctx *connContext) {
 			return
 		} else if err != nil {
 			// Normal when receive a CLOSE_REQ
-			klog.ErrorS(err, "connection read failure", "connectionID", connID)
```

Comment:
It would be unforuntate to not log this though when there's an error when CLOSE_REQ wasn't received? 

Wondering if there's a more deterministic way to only ignore the log when we know CLOSE_REQ was received.

Reply:
I'm still getting familiar with the code here, but is it an option to check the connection ID from `a.connManager` before deciding the log level?

Reply:
If CLOSE_REQ has been called, then the cleanFunc will have been called. In the cleanFunc we remove connID from the connManager prior to closing the conn. So you should be able to use the presence of connID in the a.connManager to determine the log level.

Reply:
^ @mainred can we make this change please?